content/en/boilerplates/trust-config.md

```diff
@@ -0,0 +1,45 @@
+---
+---
+For `CLUSTER_2` to participate in cross-cluster load balancing with your
+first cluster, in this case `CLUSTER_1`, establish trust
+between the clusters and generate a Certificate Authority (CA) certificate for
+`CLUSTER_2` that the common root CA signed.
+Using the set environment variables, configure trust with the following steps:
+
+1. Go to the `${WORK_DIR}` folder with the following command:
+
+{{< text bash >}}
+$ cd ${WORK_DIR}
+{{< /text >}}
+
+1. Generate the intermediate CA files for `Cluster_2` with the following command:
+
+{{< text bash >}}
+$ make -f ${ISTIO}/tools/certs/Makefile ${CLUSTER_2}-cacerts-k8s
```

Comment:
This does not seem to work:

```
make -f $ISTIO/tools/certs/Makefile cluster1-cacerts-k8s    ✘ 130 
fetching root ca from k8s cluster: gke_landow-istio-dev_us-central1-c_cluster-1
Error from server (NotFound): secrets "istio-ca-secret" not found
Error from server (NotFound): secrets "istio-ca-secret" not found
generating cluster1/ca-key.pem
Generating RSA private key, 4096 bit long modulus (2 primes)
......................................................................++++
....................++++
e is 65537 (0x010001)
generating cluster1/cluster-ca.csr
generating cluster1/ca-cert.pem
Signature ok
subject=O = Istio, CN = Intermediate CA, L = cluster1
unable to load certificate
4687273408:error:0909006C:PEM routines:get_name:no start line:crypto/pem/pem_lib.c:745:Expecting: TRUSTED CERTIFICATE
make: *** [cluster1/ca-cert.pem] Error 1
rm cluster1/cluster-ca.csr cluster1/intermediate.conf
```

I noticed the makefile has some kubectl usages without context. Wondering if that's an issue when setting up additional clusters? 

The rest of these docs I've run through successfully while just using the pre-generated samples certs. 

cc @nmittler